tests/tui/test_tui_coverage_gaps.py

```
@@ -0,0 +1,667 @@
+        def _query_one(panel_type):
+            mapping = {
+                MethodologySelectorPanel: mock_selector,
+                MethodologyPreviewPanel: mock_preview,
+            }
+            return mapping[panel_type]
+
+        view.query_one = MagicMock(side_effect=_query_one)
+        return view, mock_selector, mock_preview
+
+    def test_action_set_para(self):
+        view, selector, preview = self._make_view()
```

Ruff (0.15.18)

[warning] Unpacked variable `preview` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)